float32[3] accel
float32[3] gyro
float32[4] quat
